packages/components/spec/components/dropdown/Dropdown.spec.tsx

```diff
@@ -108,6 +108,39 @@ describe('Dropdown component test suite =>', () => {
         expect(onClear).toBeCalled();
         expect(getByText('Select...')).toBeTruthy();
       });
+
+
+      it('should support custom option structure', async () => {
```

Comment:
interesting that you had the idea of adding this test case in dropdown
What was your motivation?